lesson_08/script.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+const ul = document.getElementById("js-list");
+const newElement = document.createElement("div");
+newElement.id = "js-loading";
+ul.before(newElement);
+
+// ローディング作る
+const addLoading = () => {
+  const loading = document.getElementById("js-loading");
+  const gif = document.createElement("img");
+  gif.src = "loading-circle.gif";
+  loading.appendChild(gif);
+};
+// ローディング消す
+const removeLoading = () => document.getElementById("js-loading").remove();
+// リスト
+const getData = new Promise((resolve, reject) => {
+  addLoading();
+  const data = [
+    { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+    { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+  ];
+  setTimeout(() => {
+    resolve(data);
```

Comment:
```suggestion
    resolve();
```
The purpose is to reject it, so I don't dare to pass data.